src/queries.ts

```diff
@@ -1,18 +1,25 @@
-import { gql } from "graphql.macro";
+import { graphql } from "./common/graphql/types";
 
-const COUNT_FRAGMENT = gql`
+export const CountCharactersFragment = graphql(`
   fragment Count on Characters {
     info {
       count
     }
   }
-`;
+`);
 
-export const GET_CHARACTER = gql`
-  ${COUNT_FRAGMENT}
+export const CharactersCountFragment = graphql(`
```

Comment:
We need to create a fragment specific for `<CharactersCount>`.

This is the data requirements of `<CharactersCount>` and that's what we'll use to unmask the data from the query.